### PR TITLE
[Bugfix/LIVE-11132]: add ptxSwapMoonpayProviderEnabled as an identity and tracker analytics property

### PR DESCRIPTION
**Pull request by kallen-ledger**
### 📝 Description

- add ptxSwapMoonpayProviderEnabled as an identity and tracker analytics property.
- remove from swap page since it is now an extra property for all events.

### ❓ Context

https://ledgerhq.atlassian.net/browse/LIVE-11132

### ✅ Checklist

Pull Requests must pass the CI and be code reviewed. Set as Draft if the PR is not ready.

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [x] **Impact of the changes:** 

---

### 🧐 Checklist for the PR Reviewers

- [ ] **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- [ ] **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- [ ] **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- [ ] **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- [ ] **Any new dependencies** have been justified and documented.
- [ ] **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

**Review by beths-ledger — DISMISSED** [review]

**Review by beths-ledger — COMMENTED** [review]
Do we want this on LLM as well? Or just desktop?

**Comment by kallen-ledger** [comment]
> Do we want this on LLM as well? Or just desktop?

@beths-ledger This is good for desktop only.

File: apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/utils/index.ts [hunk 1]
```diff
@@ -3,19 +3,16 @@ import { useHistory } from "react-router-dom";
...
 export const useGetSwapTrackingProperties = () => {
-  const ptxSwapMoonpayProviderFlag = useFeature("ptxSwapMoonpayProvider");
   return useMemo(
     () => ({
       swapVersion: SWAP_VERSION,
       flow: "swap",
-      ptxSwapMoonpayProviderEnabled: !!ptxSwapMoonpayProviderFlag?.enabled,
     }),
-    [ptxSwapMoonpayProviderFlag?.enabled],
+    [],
```
Comment (KVNLS) [inline apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/utils/index.ts:15]: As now it's a const I think that you can extract the object outside the hook and return the newly created const object from the hook, so you don't need to memo, wdyt?

**Review by KVNLS — CHANGES_REQUESTED** [review]

File: apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/utils/index.ts (hunk 1 above)
Comment (kallen-ledger) [reply apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/utils/index.ts:15]: Yup, agreed 👍

**Review by KVNLS — APPROVED** [review]

**Review by beths-ledger — APPROVED** [review]

**Pull request merged**